daemon/mgr/spec_linux.go

```diff
@@ -69,21 +69,18 @@ func populatePlatform(ctx context.Context, c *Container, specWrapper *SpecWrappe
 	// setup something depend on privileged authority
 	if !c.HostConfig.Privileged {
 		s.Linux.MountLabel = c.MountLabel
+
+		// if MaskedPaths or ReadonlyPaths are set, we will use them, otherwise using the default values.
+		if len(c.HostConfig.MaskedPaths) > 0 {
```

Comment:
check is nil here? because if user passes the empty array here? should we care this?

Reply:
both no-pass and pass-empty are the same. synced with @HusterWan

Reply:
if we do not pass this parameter, we will generate a default value. 
refer: https://github.com/alibaba/pouch/blob/master/oci/spec_default.go#L88